src/View/layout.html.twig

```diff
@@ -5,22 +5,21 @@
 		<meta http-equiv="x-ua-compatible" content="ie=edge">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<link rel="icon" href="/assets/images/icons/favicon.png">
+
+		{% block stylesheet%}
+		<link rel="stylesheet" href="/assets/css/blur.css">
 		<link rel="stylesheet" href="/assets/css/style.css">
```

Comment:
global css should be outside stylesheet block, as well for header and footer if those components are available on all website